### PR TITLE
Add bugfix release 2.6.7

### PR DESCRIPTION
**Pull request by dhollinger**

### CODE REVIEW

File: org.freeciv.Freeciv.json [hunk 1]
```diff
@@ -1,44 +1,45 @@
 {
-    "app-id": "org.freeciv.Freeciv",
```
Comment (hfiguiere) [inline org.freeciv.Freeciv.json]: please don't rewrite the whole manifest because you don't like the indentation.

Comment (dhollinger) [reply org.freeciv.Freeciv.json]: That wasn't done intentionally, it was the JSON language server auto-formatting on save in NeoVim

Comment (dhollinger) [reply org.freeciv.Freeciv.json]: indentation should be fixed now

**Comment by hfiguiere** [comment]
thanks!

**Pull request merged**